machy8/webloader-bundle/1.1/config/packages/web_loader.yaml

```diff
@@ -0,0 +1,4 @@
+web_loader:
+    pathPlaceholders:
+    filesCollections:
```

Comment:
Why are these needed? Do they do anything?

Reply:
Yes. Path placeholders and files collections are the most used configuration parameters in the webloader. So they are here, prearranged, yet empty.

Reply:
be careful, they are `null` here, not an empty string, due to the way Yaml works. The empty string must be represented as a quoted value. So make sure your bundle handles null values, or update this.

Reply:
Weloader have condition check for the null values https://github.com/Machy8/webloader-bundle/blob/master/DependencyInjection/WebLoaderExtension.php#L43